### PR TITLE
Update ty page marketing component

### PR DESCRIPTION
**Pull request by walaura**
## Why are you doing this?

It was rendering quite poorly and looked bad, this fixes it. In doing so it updates the marketing consent container button to use `productPageButton` instead, so to reuse the styles.

[**Trello Card**](https://trello.com/c/XSqJvW5F/2142-fix-checkout-ty-page-layout-issues)

## Screenshots
support thegulocal com_uk_subscribe_digital_checkout_displaycheckout true
screenshot 2019-01-14 at 2 28 20 pm

### CODE REVIEW

File: assets/components/productPage/productPageButton/productPageButton.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@
```
Comment (jranks123) [inline assets/components/productPage/productPageButton/productPageButton.jsx]: What does product page button refer to? Is that the name of the "showcase page"? In which case, maybe we should call this something more general as it's now being used on the contributions flow?

File: assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.scss [hunk 2]
```diff
@@ -126,3 +104,21 @@
...
 }
 
+.component-marketing-consent.component-marketing-consent--newsletter {
+  border-bottom: 1px solid gu-colour(garnett-neutral-4);
+}
+
+.component-marketing-consent__message {
+  font-family: $gu-text-egyptian-web;
+  letter-spacing: -0.5px;
+  margin-bottom: 24px;
```
Comment (jranks123) [inline assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.scss]: You could use ``$gu-v-spacing` * 2` here

File: assets/components/productPage/productPageButton/productPageButton.jsx (hunk 1 above)
Comment (walaura) [reply assets/components/productPage/productPageButton/productPageButton.jsx]: this is why I wanna have this chat on buttons, to sort out how we wanna unify buttons. At the moment i just did this to unblock the checkout landing so we didn't have to reuse styles

File: assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.scss (hunk 2 above)
Comment (walaura) [reply assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.scss]: I copypasted the current code :D happy to do this in a subsequent PR

Comment (jranks123) [reply assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.scss]: Would that not be overkill? A whole PR to change `24px` to `$gu-v-spacing * 2`? Up to you, but I would recommend cleaning up as you go along here!

**Pull request merged**